### PR TITLE
Created a script to generate content template Excel and updated emails

### PR DESCRIPTION
**Pull request by euanmillar**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -29,4 +29,5 @@ src/tests/locations.json
 cypress/screenshots
 cypress/videos
 graphql.schema.json
-*.tar.gz
\ No newline at end of file
+*.tar.gz
+9. config: Translations
```
Comment (rikukissa) [inline .gitignore]: The gitignore doesn't seem to work here :/ maybe `*.xlsx` would suffice?

Comment (euanmillar) [reply .gitignore]: Yeah will do.  This was also because I removed the extension in the filename that actually goes on Drive.

**Comment by euanmillar** [comment]
@rikukissa this is the script I wrote to create the content files that we could add to the publish release pipeline somehow to generate a new content Excel.

**Comment by rikukissa** [comment]
Cool 👍 should we move this to core and add the pipeline there? Seems there is already a `yarn extract:translations` in client, config, login and notification. Maybe we could leverage those and create a script to the top level package.json

**Comment by euanmillar** [comment]
@rikukissa I think this can be closed as we are migrating to CSVs in this PR right? https://github.com/opencrvs/opencrvs-farajaland/pull/843

**Comment by rikukissa** [comment]
@euanmillar closing this in favour of the approach [here](https://github.com/opencrvs/opencrvs-farajaland/pull/843). Let's reopen if needed

**Pull request closed without merging**